File: app/src/main/java/com/diploma/bank/Authorization/UserAccountControl.java
package com.diploma.bank.Authorization;

import com.diploma.bank.TempData.UserAccounts;

public class UserAccountControl {
    public static boolean CheckUser(String login, String password){
        String[] login_accounts = new String[UserAccounts.user_account.length];
        for (int i = 0; i < login_accounts.length; i++) {
            login_accounts[i] = UserAccounts.user_account[i].getLogin();
            if (login.equals(login_accounts[i])){
                if (password.equals(UserAccounts.user_account[i].getPassword())){
                    return true;
                }
                else break;
            }
        }
        return false;
    }
}

File: app/src/main/java/com/vmbeshenov/bank/Authorization/UserAccountControl.kt
package com.vmbeshenov.bank.Authorization

import com.vmbeshenov.bank.TempData.UserAccounts

object UserAccountControl {
    fun checkUser(login: String, password: String): Boolean {
        val loginAccounts = arrayOfNulls<String>(UserAccounts.user_account.size)
        for (i in loginAccounts.indices) {
            loginAccounts[i] = UserAccounts.user_account[i].login
            if (login == loginAccounts[i]) {
                return if (password == UserAccounts.user_account[i].password) {
                    true
                } else break
            }
        }
        return false
    }
}